### PR TITLE
Move to Circle [changelog skip]

### PR DESCRIPTION
**Pull request by danielleadams**
Moving tests to Circle Ci.

### Includes
- Added Circle CI configuration file
- Replace badge on README
- Update Hatchet scripts to work with CI
- Remove Travis CI configuration file
- Remove cedar-14 from Makefile

Follow up:
- Make a PR to update shellcheck tests

### CODE REVIEW

**Review by edmorley — COMMENTED** [review]
The Circle tests are failing with:
`^@^@bash: test/unit: No such file or directory`

Marking the PR as draft for now

File: makefile [hunk 1]
```diff
@@ -36,11 +36,6 @@ heroku-16-build:
 	@docker run -v $(shell pwd):/buildpack:ro --rm -it -e "STACK=heroku-16-build" heroku/heroku:16-build bash -c 'cp -r /buildpack /buildpack_test; cd /buildpack_test/; test/run;'
 	@echo ""
 
-cedar-14:
```
Comment (edmorley) [inline makefile:39]: I think the `test` target in this file needs updating to remove the `cedar-14` reference

**Review by edmorley — APPROVED** [review]
🎉

File: makefile (hunk 1 above)
Comment (danielleadams) [reply makefile:39]: oh good catch! updated.

**Pull request merged**